app/javascript/components/enrollment/Enrollment.js

```diff
@@ -184,17 +187,28 @@ class Enrollment extends React.Component {
         this.setState({ index: lastIndex, lastIndex: null });
       });
     } else {
+      let step = 1;
+      if ((this.state.enrollmentState.isolation ? 4 : 5) == index) {
+        step = 2;
+      }
       this.setState({ direction: 'next' }, () => {
-        this.setState({ index: index + 1, lastIndex: null });
+        this.setState({ index: index + step, lastIndex: null });
       });
+      if (index + step == maxEnrollmentSteps) {
```

Comment:
can we add this to the other set state? something like `reviewing: index + step == maxEnrollmentSteps`

Reply:
thoughts on this?

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/975/commits/c1f73425905c43de6a0cb4c0db5da986774dec31